en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -11,6 +11,9 @@ In the open data context project, it is packaged independently through the open
 
 __Cocos Creator__ supports packaging to open data contexts since v1.9.1, and an important update in v2.0.1. The two versions are used differently, as described below.
 
+`Note`: Since the subsequent version of WeChat games will support the webgl rendering mode for open data context, Creator's open data context has added support for webgl in advance, resulting in the current open data context encountering the [[GameOpenDataContext] subdomain only supported. Using 2D rendering mode's error message,
```

Comment:
**Node**: Because WeChat Mini Games will support WebGL rendering mode for Open Data Context in later versions, so Creator adapted WebGL mode for Open Data Context in v2.0.9. However, it currently cause the project to appear **[GameOpenDataContext] Open Data Context only supports using 2D rendering mode** error message when running in the WeChat developer tool.  This error message is due to the use of `document.createElement("canvas").getContext("webgl")` to detect if WeChat mini games support WebGL, it will not affect the normal use of the project, you can ignore it.